pdp/handlers.go

```diff
@@ -181,7 +181,7 @@ func (p *PDPService) handleCreateProofSet(w http.ResponseWriter, r *http.Request
 	}
 
 	// Pack the method call data
-	data, err := abiData.Pack("createProofSet", recordKeeperAddr)
+	data, err := abiData.Pack("createProofSet", recordKeeperAddr, []byte{})
```

Comment:
What's this last param? We may want to allow the caller (which is the service like Storacha) to set it

Reply:
Yeah that sounds about right.  The last param is the one we talked about in [this issue](https://github.com/FilOzone/pdp/issues/56).  I didn't want to mess with the HTTP api in this PR but adding extra data field to all of the user facing methods should probably happen.